netty/src/test/java/io/grpc/netty/NettyChannelBuilderTest.java

```diff
@@ -90,6 +93,24 @@ public void failInvalidAuthority() {
   }
 
   @Test
+  public void sslContextCanBeNull() {
+    NettyChannelBuilder builder = mock(NettyChannelBuilder.class);
+    builder.sslContext(null);
```

Comment:
This test won't do anything since mocks will not throw on an invalid param.  You'll need to make a real NCB to test with.